openhtf/names.py

```diff
@@ -55,5 +56,6 @@
 # Classes used in writing test scripts.
 Measurement = openhtf.util.measurements.Measurement
 OutputToJSON = openhtf.io.output.json_factory.OutputToJSON
+UploadJSON = openhtf.io.output.upload_json.UploadJSON
```

Comment:
This upload_json is not centered to uploading to any "CM", it is a general loader to any data server. uploading test results to a server is a  necessary step to complete the whole circle of testing. It should have the same weight as OutputToJSON.